src/assets/css/site.css

```
@@ -3310,14 +3612,65 @@ h1 {
+  transform-origin: center;
+  transition: opacity 320ms ease, transform 500ms cubic-bezier(0.4, 0, 0.2, 1);
 }
 
 .workflow-related__hero.is-loaded::after {
   opacity: 1;
 }
 
+.workflow-related__placeholder-icon {
+  --icon-stroke-width: 1.25;
+  --placeholder-icon-hover-rotation: -4deg;
+  color: var(--color-text-muted);
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)